packages/api-client/package.json

```diff
@@ -28,11 +28,12 @@
   },
   "dependencies": {
     "@8base/error-codes": "^1.0.5",
+    "@8base/webpack-configuration": "0.3.5",
```

Comment:
i think webpack-configuration should be a dev dependency.

Reply:
fixed

Reply:
fixed